plugins/modules/ini_file.py

```diff
@@ -116,6 +116,13 @@
       - Allow option without value and without '=' symbol.
     type: bool
     default: false
+  modify_inactive_option:
+    description:
+      - Do not replace a commented line that matches the given option. This is useful when you want to keep example
```

Comment:
I think the description says what happens if this is set to `false`, which is somewhat confusing; for boolean options the description usually says what happens if you set it to `true`. How about something like:
```suggestion
      - By default the module replaces a commented line that matches the given option.
      - Set this option to V(false) to avoid this. This is useful when you want to keep example.
```

Reply:
I agree with you on this. I'm wondering though, what's the purpose of using this syntax: `V(false)` and using double backticks (``)?

Reply:
`V()` is Ansible semantic markup, see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation.

In the changelog fragment, ReStructured Text (RST) syntax is used, which uses double backticks for literals: https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup